jobs/create_manifest/create_manifest.sh

```diff
@@ -1,17 +1,23 @@
 #!/bin/bash
 set -ex
+
+if [  -z "$MANIFEST_TYPE" ]; then
```

Comment:
Why not directly pass the template manifest file as a parameter to this script ?